README.md

```diff
@@ -1,56 +1,38 @@
 producing complex, multi-container applications.
 
-## Quick Start
+The vagrant-service-manager provides the user with:
 
-1. Install `vagrant-service-manager` plugin:
+* A CLI to configure the ADB for different use cases and to provide an interface between ADB and the user's development environment.
 
-        vagrant plugin install vagrant-service-manager
-
-2. Get a Vagrantfile for your box. Users of the
-[Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) should download a [Vagrantfile from the repository](https://github.com/projectatomic/adb-atomic-developer-bundle/tree/master/components).
-
-3. Enable your desired service(s) in [Vagrantfile](Vagrantfile) as:
-
-        config.servicemanager.services = 'openshift'
-
-   *Note*
+* A tool to control and configure the ADB from the
+developer's workstation without having to `ssh` directly into the ADB virtual machine.
 
-   * `docker` is default service and does not require above configuration.
-   * Enable multiple services as comma separated list. Eg: 'docker, openshift'
 
-4. Enable any specific options for the services you have selected:
+## Example execution of the plugin
 
-    * OpenShift
+1. Install vagrant-service-manager plugin:
 
-        Specific versions can be specified using the following variables:
-
-        `config.servicemanager.openshift_docker_registry = "docker.io"` - What registry should be pulled from
-        `config.servicemanager.openshift_image_name = "openshift/origin"` - What image should be used
-        `config.servicemanager.openshift_image_tag = "v1.1.1"` - What image version should be used
+        vagrant plugin install vagrant-service-manager
 
+2. Get a Vagrantfile for your box of choice. Users of the
+[ADB](https://github.com/projectatomic/adb-atomic-developer-bundle) should download the relevant [Vagrantfile from the repository](https://github.com/projectatomic/adb-atomic-developer-bundle/tree/master/components/centos). For further details and to use custom vagrant files designed for specific use cases, refer to the [usage documentation](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/using.rst).
 
-5. Start the ADB using `vagrant up`. Users of the ADB may wish to consult the
+3. Start the vagrant box or ADB using `vagrant up`. For detailed instructions consult the
 [Installation Documentation](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst).
 
-6. Run the plugin to get environment variables and certificates:
+	**Note:** When the vagrant-service-manager plugin is loaded and a box is started using the VirtualBox provider, the user needs to add a routable non NAT network interface declaration in the Vagrantfile. If the user does not provide a network declaration in the Vagrantfile, a private DHCP network is added by default and a warning is displayed.
+
+4. Run the plugin to get environment variables and certificates:
```

Comment:
@bexelbie : for this PR, do we want to have just one sample run of command? and we plan to include the different examples of command execution with elaboration?